<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_user_data</class>
 <widget class="QDialog" name="Add_user_data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Add_save">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_cancle">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Add_money">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>04b</family>
     <pointsize>16</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;04b&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="save_in">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>160</y>
     <width>51</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>收入</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="save_out">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>175</y>
     <width>51</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>支出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="time">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>04b</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </widget>
  <widget class="QLabel" name="movie">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>294</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/sources/cow.gif);</string>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Add_backup">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>294</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>384</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/login/images/back.png);</string>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </widget>
  <widget class="QComboBox" name="type">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文隶书</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string>支付事项</string>
   </property>
  </widget>
  <widget class="QComboBox" name="money_type">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文隶书</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string>支付方式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>记录号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_number">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>Add_save</zorder>
  <zorder>Add_cancle</zorder>
  <zorder>time</zorder>
  <zorder>Add_backup</zorder>
  <zorder>movie</zorder>
  <zorder>type</zorder>
  <zorder>money_type</zorder>
  <zorder>save_in</zorder>
  <zorder>Add_money</zorder>
  <zorder>save_out</zorder>
  <zorder>label</zorder>
  <zorder>text_number</zorder>
  <zorder>close</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
